maro/simulator/scenarios/cim/business_engine.py

```diff
@@ -510,19 +523,20 @@ def _on_full_load(self, evt: Event):
         vessel.early_discharge = 0
 
         if total_container * container_volume > vessel.capacity:
-            early_discharge_number = total_container - ceil(vessel.capacity / container_volume)
+            early_discharge_number = total_container - \
```

Comment:
will ``            early_discharge_number = total_container - ceil(vessel.capacity / container_volume)`` exceeds the length limit?

if yes, use the block below instead?
```
            early_discharge_number = (
                total_container - ceil(vessel.capacity / container_volume)
            )
```

or, further:
```
            early_discharge_number = (
                total_container
                - ceil(vessel.capacity / container_volume)
            )
```